DevelopedCalculator.js

```diff
@@ -27,12 +27,16 @@ class DevelopedCalculator extends Calculator {
     gettingHistoryArray() {
         // updating the history element by the history which is joined with ,
         this.historyElement.textContent = this.history.join(",")
+            //  this.historyElement.textContent = this.history.join("")
     };
     onOperatorButtonClick(event) {
+
             // calling parent class's onOperatorButtonClick method
             super.onOperatorButtonClick(event);
             const curntresultandopr = this.currentResult + this.operator
             this.history.push(curntresultandopr);
+            this.operator = null;
```

Comment:
Explain your changes pls

Reply:
this.oper = null
If(..

Here I am stopping the operator from calculating because I want to see the current result only in history
Not the calculation

Reply:
> this.oper = null
> If(..
> 
> Here I am stopping the operator from calculating because I want to see the current result only in history
> Not the calculation

what do you mean by calculation?

Reply:
> > this.oper = null
> > If(..
> > Here I am stopping the operator from calculating because I want to see the current result only in history
> > Not the calculation
> 
> what do you mean by calculation?

Adding or multiplying or dividing or subtracting

Reply:
You mean If I want to click on these buttons 
2 + 2

you want to see in in the history element  2,+,2
and in the result element 4?

but no one clicked on the = button yet?

what if a user want to do 
2 + 22 ?.

Reply:
Yes
No need to click the = 
Because we updated the old implementation to perform calculation by clicking on operator buttons